data/drules_proto_outdoors_clear.txt

```diff
@@ -12395,7 +12395,7 @@ cont {
   element {
     scale: 13
     symbol {
-      name: "shelter-s"
+      name: "shelter-public_transport-m"
```

Comment:
either we need an `-s` icon for z13,
or (better) let's make PT shelters  z14- in outdoors style (you'll have to reset the default icon on z13)

Reply:
I've removed it from z13